### DISCUSSION TITLE
ıbax puan nasıl artırılır

### DISCUSSION THREAD

**Discussion by Kriptoportakal** [IBAX-io/go-ibax#657] [Soru Cevap - Türkçe]
ıbax telegram ve discort kanallarına girene 1000 point veriyor mu alan var mı aldıysanızda nasıl claim edicez?

**Comment by Yusufknk** [comment]
Bu puanlar 1 Şubat'tan önce ayrı bir sayfa/formda ilan edilecek. Ayrıca bu süreye kadar Sandıktan, retweet görevlerinden ve çekilişlerden puanlar kazanabilirsiniz. Tüm üyeler topladığı puana göre coin alacaklar. 1 Şubat'tan sonra dağıtıma başlanacak.
